hazelcast/src/main/java/com/hazelcast/map/impl/NearCacheSizeEstimator.java

```diff
@@ -45,12 +46,9 @@ public long getCost(NearCacheRecord record) {
         if (cost == 0) {
             return 0;
         }
-        final int numberOfIntegers = 4;
-        long size = 0;
         // entry size in CHM
-        size += numberOfIntegers * ((Integer.SIZE / Byte.SIZE));
-        size += cost;
-        return size;
+        final int numberOfIntegers = 4;
+        return numberOfIntegers * (Integer.SIZE / Byte.SIZE) + OBJECT_HEADER_IN_BYTES + cost;
```

Comment:
minor comment: `Bits.INT_SIZE_IN_BYTES` is easier to read, instead of `(Integer.SIZE / Byte.SIZE)`.